tests/PermissionTest.php

```diff
@@ -41,4 +41,16 @@ public function it_has_user_models_of_the_right_class()
         $this->assertTrue($this->testUserPermission->users->first()->is($this->testUser));
         $this->assertInstanceOf(User::class, $this->testUserPermission->users->first());
     }
+
+    /** @test */
+    public function it_can_give_and_revoke_multiple_permissions()
```

Comment:
Can we move this test to the `HasPermissionsTest`? I think that'd be a more appropriate context